step/serialization.md

```diff
@@ -495,24 +482,26 @@ public class ClassSerializationServer {
 }
 </copy>
 ```
-In `ClassSerializationServer` `SerializationFilter` is defined that will work down the tree of any incoming serialized data for the `Runnable` type and if found, reject it. 
+
+In `SerializationServer`, a `SerializationFilter` is defined that will work down the tree of any incoming serialized data looking for the `Runnable` type and if found, reject it. 
```

Comment:
Given that the exploit section is gone, just explain that some exploit are using Runnable so we want our filter to filter them out.